.vscode/launch.json

```diff
@@ -21,7 +21,7 @@
             "type": "java",
```

Comment:
This file probably shouldn't be in the git repository at all (it's your local IDE configuration), and instead the `.gitignore` should be configured so that git ignores it.